device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/mqtt/MqttIotHubConnection.java

```diff
@@ -42,7 +42,7 @@
     private static final String SSL_PREFIX = "ssl://";
     private static final String SSL_PORT_SUFFIX = ":8883";
 
-    private static final String ModelIdParam = "digital-twin-model-id";
+    private static final String ModelIdParam = "model-id";
 
```

Comment:
Will there be separate PR for Service Client?

Reply:
There is already a PR for it.

Reply:
nm. Just saw this https://github.com/Azure/azure-iot-sdk-java/pull/834/files